src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/CarinaDescriptionPage.java

```diff
@@ -20,6 +20,26 @@ public CarinaDescriptionPage(WebDriver driver) {
 		super(driver);
 	}
 
+	@Override
+	public WebViewBase navitateToWebview() {
+		return null;
+	}
+
+	@Override
+	public ChartsBase navigateToCharts() {
+		return null;
```

Comment:
- Please use instead of "return null" something like  "throw new UnsupportedOperationException(METHOD_ISN_T_IMPLEMENTED_FOR_ANDROID_MESSAGE);"